File: GO-BASIC/09.Advanced-Loops-Lab/03.sum-numbers/sum-numbers.go
package main

import "fmt"

func main() {
	var number,currentNums, sum int
	fmt.Scanln(&number)
	fmt.Scanln(&currentNums)

	for sum < number {
		sum += currentNums

		fmt.Scanln(&currentNums)
		
	}

	fmt.Println(sum)
}

File: JavaScript/JS-Advanced/01.DOM-Introduction/03.Sum-Numbers/SumNumbers.js
function calc() {
  
  let firstElement = document.getElementById('num1').value;
  let secondElement = document.getElementById('num2').value;
  let inputSumElement = document.getElementById('sum');
  
  let res = Number(firstElement) + Number(secondElement);
  inputSumElement.value = res;
  
  
}
